templates/pages/amp/category.html

```diff
@@ -20,7 +20,7 @@
     {{> components/amp/common/header }}
     {{#if category.image}}
         <div class="categoryView-img">
-            <amp-img src="{{getImage category.image 'zoom_size'}}" width="300" height="150" layout="responsive" alt="{{category.name}}"></amp-img>
+            <amp-img srcset="{{getImageSrcset category.image use_default_sizes=true}}" layout="responsive" alt="{{category.name}}"></amp-img>
```

Comment:
deleting dimensions as they're not necessary with `layout="responsive"` and a `srcset` 👍